crypto/bio/b_print.c

```diff
@@ -767,7 +767,6 @@ int BIO_vprintf(BIO *bio, const char *format, va_list args)
     int ignored;
 
     dynbuf = NULL;
-    CRYPTO_push_info("doapr()");
```

Comment:
I decided that it’s really only needed for recursive functions, like the ASN1 stuff.

From: kaduk [mailto:notifications@github.com]
Sent: Tuesday, November 17, 2015 6:29 PM
To: openssl/openssl
Cc: Salz, Rich
Subject: Re: [openssl] mem functions cleanup (#450)


In crypto/bio/b_print.c<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_openssl_openssl_pull_450-23discussion-5Fr45138797&d=CwMCaQ&c=96ZbZZcaMF4w0F4jpN6LZg&r=4LM0GbR0h9Fvx86FtsKI-w&m=vJ8NZvmEA4skkelptUpiaB4aXtOndSk3KhMUN_X9DUo&s=wNLYmvSBM9gY9sEgShlwRiMygo6j2RacIKDqfCP368Y&e=>:

> @@ -767,7 +767,6 @@ int BIO_vprintf(BIO *bio, const char *format, va_list args)

>      int ignored;

>

>      dynbuf = NULL;

> -    CRYPTO_push_info("doapr()");

I'm not sure I understand why this is being removed -- _dopr() appears to have a codepath that allocates memory (via doapr_outch()). On the other hand, I'm not sure I could provide an argument for why it should be present, either.

—
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_openssl_openssl_pull_450_files-23r45138797&d=CwMCaQ&c=96ZbZZcaMF4w0F4jpN6LZg&r=4LM0GbR0h9Fvx86FtsKI-w&m=vJ8NZvmEA4skkelptUpiaB4aXtOndSk3KhMUN_X9DUo&s=yXUnsLLH-dv-Mv9j4p8hQXhalmJjpQBhQQ2QQ0wfVjo&e=>.